server/util/ipfs.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
We should make this filename more descriptive, as I assume we will, over time, have significantly more ipfs-related infrastructure. I would rename the file and function to `pinIpfsBlob`.

Reply:
Done